devops/cts_exclude_filter

```diff
@@ -0,0 +1,52 @@
+accessor
+accessor_legacy
+address_space
+atomic
```

Comment:
I hope you are going to enable all passing tests in follow-up PRs. Right?

Reply:
Yes. of course.